.travis.yml

```diff
@@ -7,20 +7,14 @@ rust:
 matrix:
   allow_failures:
     - rust: nightly
+  fast_finish: true
 
 jdk:
   - oraclejdk8
```

Comment:
For `lang: rust` it has no effect, doesn't it? (I remember having bumped into this earlier: #68 )

Reply:
Hm, it's present at [the Travis page about Rust page](https://docs.travis-ci.com/user/languages/rust/#choosing-a-rust-version)

> I remember having bumped into this earlier: #68

I'm afraid I don't remember such issue. Well, I feel like it worked as expected when I tested this config on separate branch and it used to fail.

Reply:
`jdk` appears to have no effect on the actual JDK installed, and it is not documented on that page.